### PR TITLE
Add "Why Polar Signals"

### PR DESCRIPTION
**Pull request by brancz**

### CODE REVIEW

**Comment by brancz** [comment]
Note to self: Add section on strong Kubernetes integration

File: docs/why-polar-signals.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+### Filter by function
+
+You already know what you want to optimize? Great! Use the filter-by-function functionality
+
+### Compare
+
+Polar Signals Cloud allows comparing the result of any two queries against each other. Compare the CPU time of version X with version Y (to understand a performance regression), compare datacenter A with datacenter B, or just understand what code was executing at one point in time of a process vs. another point in time.
+
+## Polar Signals Cloud vs. Parca
```
Comment (thorfour) [inline docs/why-polar-signals.md]: I would maybe roll this up into the top why polar signals cloud section. Get it out of the way first that this is a hosted version of Parca and it scales with your business without having to maintain the software etc.

Comment (brancz) [reply docs/why-polar-signals.md]: Sounds good!

**Review by kakkoyun — COMMENTED** [review]
Looking good.

Friendly reminder: We might want to add the docs to the sidebar.

**Pull request merged**